draft-irtf-cfrg-opaque.md

```diff
@@ -770,25 +767,48 @@ Steps:
 
 #### CreateCredentialResponse {#create-credential-response}
 
+There are two scenarios to handle for the construction of a CredentialResponse object: either the
+credential_file record for the client exists (corresponding to a properly registered client), or
+it was never created (corresponding to a client that has yet to register).
+
+In the case of an existing credential_file record with the corresponding identifier
+`credential_identifier`, the server invokes the following function to
+produce a CredentialResponse:
+
 ~~~
-CreateCredentialResponse(request, server_public_key, credential_file)
+CreateCredentialResponse(request, server_public_key, credential_file, credential_identifier, oprf_seed)
 
 Input:
 - request, a CredentialRequest structure
 - server_public_key, the public key of the server
-- credential_file, the server's output from registration
-  (see {{credential-file}})
+- credential_file, an instance of RegistrationUpload which is the server's
```

Comment:
```suggestion
- record, an instance of RegistrationUpload which is the server's
```

Reply:
(here and elsewhere, since we refer to the element of this type as `record` in the FinalizeRequest function.)

Reply:
Done!